src/ui/include/nqqsettings.h

```diff
@@ -63,6 +63,7 @@ class NqqSettings {
 public:
 
     BEGIN_GENERAL_CATEGORY(General)
+        NQQ_SETTING(FirstRun,                       bool,       true)
         NQQ_SETTING(Localization,                   QString,    "en")
```

Comment:
Yea, but I was thinking about other settings that might need to be initialized in the future on a first run basis... if you don't think this is an issue I can default it to empty instead.